pkg/cfn/builder/api.go

```diff
@@ -30,6 +30,7 @@ type ResourceSet interface {
 	WithNamedIAM() bool
 	RenderJSON() ([]byte, error)
 	GetAllOutputs(cfn.Stack) error
+	Template() gfn.Template
```

Comment:
Why not use `RenderJSON()`? I suppose the format is not too pretty, but we could add a parameter that make it prettier, like `RenderJSON(pretty bool)`...

Reply:
because gfn.Template implements JSON and YAML marshaling